go/private/go_toolchain.bzl

```diff
@@ -200,6 +200,33 @@ def declare_bazel_toolchains(
         visibility = ["//visibility:private"],
     )
 
+    sdk_name_label = Label("//go/toolchain:sdk_name")
```

Comment:
You can inline this and drop the `Label(...)` call, `config_setting` properly converts its attributes.

Reply:
This doesn't work for me. In this case it fails with 
```
ERROR: no such package '@@go_sdk_toolchains//go/toolchain': BUILD file not found in directory 'go/toolchain' of external repository @@go_sdk_toolchains. Add a BUILD file to a directory to mark it as a package.
```

IIUIC the reason for this is that `declare_bazel_toolchains` is called from the dynamically created repo `go_toolchains`, while the setting is defined in `io_bazel_rules_go`. The `Label` constructor is resolving labels relative to where the `bzl` file resides, not relative to where it is called from.

Btw. same technique is used for the already existing `sdk_version` just above.

Reply:
You are right, thanks for the explanation!